customer-service/src/main/java/entelect/training/incubator/spring/customer/controller/CustomersController.java

```diff
@@ -66,10 +62,24 @@ public CustomersController(CustomersService customersService) {
         return ResponseEntity.notFound().build();
     }
 
-    @PostMapping("/search")
-    public ResponseEntity<?> searchCustomers(@RequestBody CustomerSearchRequest searchRequest) {
-        LOGGER.info("Processing customer search request for request {}", searchRequest);
+    @PostMapping(path = "search/{searchType}")
+    public ResponseEntity<?> searchCustomers(@PathVariable("searchType") String searchTypeString,
```

Comment:
This change seems incorrect. When doing a POST request the data should really be living in the request body. What was the reason for the change?

Reply:
Changed to match existing unit tests.

These tests are in fact incorrect and should be changed, this change will then be reverted.